File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/mad/Java/Go/s780275612_commented.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var intA int
	if _, err := fmt.Scan(&intA); err != nil {
		fmt.Println("Error reading input:", err)
		return
	}

	dbAncer := math.Pow(float64(intA), 3)
	intAncer := int(dbAncer)
	fmt.Println(intAncer)
}
// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/javadoc/uncomment/direct/Java/C/s780275612_commented.c
#include <stdio.h>
#include <math.h>

int main() {
    int intA;
    scanf("%d", &intA);
    double dbAncer = pow((double)intA, 3);
    int intAncer = (int)dbAncer;
    printf("%d\n", intAncer);
    return 0;
}

// End of Code
